guides/hosting/infrastructure/message-queue.md

```diff
@@ -55,6 +55,18 @@ shopware:
         transports: ["default"]
 ```
 
+### Sending Mails over the Message Queue
+
+By default sends Shopware the Mails synchronous. This can affect the page speed. You can switch it to use the Message Queue with an small configuration change.
```

Comment:
```suggestion
By default Shopware sends the mails synchronously. Since this can affect the page speed, you can switch it to use the Message Queue with a small configuration change.
```